viewing_party/party.py

```diff
@@ -1,23 +1,138 @@
+from statistics import mean, mode
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
-
+# -----------------------------------------
+# If valid, creates a movie with title, genre, and rating as values of a dict
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        return None
+    else:
+        movie = {"title" : title, "genre" : genre, "rating" : rating}
+    return movie

```

Comment:
What happens if `title`, `genre`, or `rating` were entered as empty strings, 0, or False? We can make your conditional more robust by checking for all _falsey_ values instead. The `else` clause is also optional here and we can omit it, allowing the creation of the movie dictionary to shine and be more in focus. 

```python
   if not title or not genre or not rating:
        return None

   movie = {"title" : title, "genre" : genre, "rating" : rating}
   return movie
```